cli/pazuzu/actions/search.go

```diff
@@ -0,0 +1,44 @@
+package actions
+
+import (
+	"errors"
+	"fmt"
+	"github.com/urfave/cli"
+	"github.com/zalando-incubator/pazuzu/config"
+	"os"
+	"regexp"
+	"text/tabwriter"
+)
+
+func Search(c *cli.Context) error {
+	if c.NArg() != 1 {
+		return errors.New("Search regexp is not provided")
+	}
+	feature := c.Args().First()
+	featureRegexp, err := regexp.Compile(feature)
```

Comment:
I see that pattern a lot in here. I understand, that it's how anyone uses golang, but maybe we can wrap it a bit, like this:

```go
featureRegexp := get_value(regepx.Compile(feature), BadRegexp)
cfg := config.GetConfig()
storage := get_value(config.GetStorageReader(*cfg), BadStorageReader)
features := get_value(storage.SearchMeta(featureRegexp), CantSearch)
``` 
and
```go
func get_value(value interface{}, err interface{}, errorType interface{}) {
    if err != nil {
        return errorType()
    } else {
        return value
    }
}
```

What do you think?